app/src/components/market/common/market_scale/index.tsx

```diff
@@ -440,10 +472,28 @@ export const MarketScale: React.FC<Props> = (props: Props) => {
       setProfitLoss((amountSharesNumber || 0) * (1 - Number(scaleBall?.value) / 100) - (amountNumber || 0))
     } else {
       if (shortShares && collateral && !isDust(shortShares, collateral.decimals)) {
-        setShortPayout((shortSharesNumber || 0) * (1 - Number(scaleBall?.value) / 100))
+        const totalShortPriceNumber = Number(formatBigNumber(totalShortPrice, collateral.decimals, collateral.decimals))
+        const shortPayoutAmount = (shortSharesNumber || 0) * (1 - Number(scaleBall?.value) / 100)
```

Comment:
Will this return NaN if scaleBall?.value is undefined?

Reply:
Yes, and it results in the slider being set to 0 which expected behaviour and actually used in the case that all liquidity has been removed from the market